src/components/IMGitHubDeploymentsDashboard/GitHubDeploymentsComponents.tsx

```diff
@@ -105,6 +105,16 @@ export const GitHubDeploymentsComponents = (props: {
     });
 
     apiDeployments.sort((a, b) => (a.id < b.id ? 1 : -1));
+
+    //build display environments for sorting
+    const distinctDeployedEnvironments = Array.from(
+      new Set(
+        apiDeployments
+          .map(m => m.environment)
+          .filter(f => !catalogEnvironments.includes(f)),
```

Comment:
This question might pertain to more than just this line of code, but it got me thinking about case insensitivity. If a team changes their environment names from title case to lowercase, it seems like we will display duplicates. I don't think that matters too much because the old deployments will eventually go away and so we'd lose the old environment names right?

Reply:
It's true environment names that are used could add to the master list of environments.  It's a little unlikely though because the deployment should get added programmatically through a workflow run.

It is possible, but then that part of the list would choose to sort the display alphabetically.  Which shouldn't be too much of a problem. Because something deployed to `Dev` being after or before `dev` would still be right.

Reply:
I also forgot to mention that the sort is done on the `[environment].toLowerCase()`, so it won't matter if they've had their cases mixed up.

https://github.com/im-open/im-github-deployments/blob/6d8da87652c9a4bcfa0044dd919d585f9ec1fc7e/src/components/Deployments/columns.tsx#L54-L55